guides/v2.3/graphql/reference/vault.md

```diff
@@ -0,0 +1,136 @@
+---
+group: graphql
+title: Vault endpoint
+---
+
+When the vault feature is enabled in the payment method configuration for Braintree, customers have the option during checkout to save their credit card information. During subsequent checkouts, the customer is presented with a list of saved payment options. If Instant Purchase is enabled, customers can even by-pass the two-step checkout process and place the order from the product page.
```

Comment:
Not only Braintree supports Vault. Also, there is might be 3rd party extensions which support Vault features. I would prefer to say: "When vault feature is supported by a payment integration and enabled, ...".

Reply:
Updated description.